### PR TITLE
implementing edit and update methods

### PR DESCRIPTION
**Pull request by nehamamandelbaum**
This PR completes issue #628 , which was to implement the edit/update methods for the notes in the new admin interface. 

How the UI looks: 

Captura de Tela 2023-11-28 às 12 26 41

### CODE REVIEW

**Review by EduardoSCosta — APPROVED** [review]

File: app/views/new_admin/notes/_form.html.erb [hunk 1]
```diff
@@ -33,7 +33,7 @@
               <%= form.select :author_id, 
                               User.pluck(:name, :id), 
                               { prompt: t('prompt'), 
-                              selected: params[:author_id] }, 
+                              selected: User.pluck(:name, :id).select {|user| user[1] == note.author_id }.first},
```
Comment (Lucas-Boeira) [inline app/views/new_admin/notes/_form.html.erb:36]: Sugestão:
                          selected: User.pluck(:name, :id).find { |user| user[1] == note.author_id } },

File: app/views/new_admin/notes/_form.html.erb [hunk 2]
```diff
@@ -50,7 +50,7 @@
                             <%= form.select :user_id, 
                               User.pluck(:name, :id), 
                               { prompt: t('prompt'), 
-                              selected: params[:author_id] }, 
+                              selected: User.pluck(:name, :id).select {|user| user[1] == note.user_id }.first },
```
Comment (Lucas-Boeira) [inline app/views/new_admin/notes/_form.html.erb:53]: Mesma sugestão acima.

File: app/views/new_admin/notes/_form.html.erb (hunk 1 above)
Comment (Lucas-Boeira) [inline app/views/new_admin/notes/_form.html.erb:36]: Sugestão:
selected: User.pluck(:name, :id).find { |user| user[1] == note.author_id } },

File: app/views/new_admin/notes/_form.html.erb (hunk 2 above)
Comment (Lucas-Boeira) [inline app/views/new_admin/notes/_form.html.erb:53]: Mesmo caso acima.

**Pull request merged**